prow/lib.sh

```diff
@@ -63,9 +63,10 @@ function download_untar_istio_release() {
 
 function build_images() {
   # Build just the images needed for tests
-  for image in pilot proxyv2 app test_policybackend mixer citadel galley sidecar_injector kubectl node-agent-k8s; do
-     DOCKER_BUILD_VARIANTS="${VARIANT:-default}" make docker.${image}
-  done
+  targets="docker.pilot docker.proxyv2 "
+  targets+="docker.app docker.test_policybackend docker.kubectl "
+  targets+="docker.mixer docker.citadel docker.galley docker.sidecar_injector docker.node-agent-k8s"
+  DOCKER_BUILD_VARIANTS="${VARIANT:-default}" DOCKER_TARGETS="${targets}" make dockerx
```

Comment:
The key here is we are modifying the targets and calling a single make target, rather than many make targets (which is slow)